src/modules/orbitdb/db-index.js

```diff
@@ -0,0 +1,63 @@
+const IPFSUtils = require('../../utils/ipfs');
+
+const version = 0;
+
+class ChluIndex {
+    constructor(){
+        this._index = {
+            reviewRecords: [],
+            updates: {}
+        };
+    }
+
+    /**
+     * Called by OrbitDB to update the index 
+     */
+    updateIndex(oplog) {
+        oplog.values
+            .slice()
+            .forEach(item => {
+                // Skip operations from a different chlu store version
+                if (item.payload.version === version) {
+                    // TODO: check update validity, RR validity if possible
+                    if (item.payload.op === operations.ADD_REVIEW_RECORD && IPFSUtils.isValidMultihash(item.payload.multihash)) {
```

Comment:
I can see we definitely want to cache validation state for a RR, esp on the service node.